src/Forms/GridField/GridField.php

```diff
@@ -696,7 +724,13 @@ public function FieldHolder($properties = [])
                 $rowContent = '';
 
                 foreach ($this->getColumns() as $column) {
-                    $colContent = $this->getColumnContent($record, $column);
```

Comment:
Note this part of the code is very close the another framework PR that is using caching for the DataObject Version number which is used for the canViewVersioned() check https://github.com/silverstripe/silverstripe-framework/pull/11710